caas/kubernetes/provider/labels_test.go

```diff
@@ -0,0 +1,49 @@
+package provider_test
+
+import (
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/juju/juju/caas/kubernetes/provider"
+)
+
+type LabelSuite struct {
+	client *fake.Clientset
+}
+
+var _ = gc.Suite(&LabelSuite{})
+
+func (l *LabelSuite) SetUpTest(c *gc.C) {
+	l.client = fake.NewSimpleClientset()
+}
+
+func (l *LabelSuite) TestIsLegacyModelLabels(c *gc.C) {
+	tests := []struct {
+		IsLegacy  bool
+		Model     string
+		Namespace *core.Namespace
+	}{
+		{
+			IsLegacy: false,
```

Comment:
there's only 1 test here and we have a slice of tests
i think you meant to add another test with IsLegacy: true